### PR TITLE
[Doc] add doc for DGLCSVDataset

### PR DESCRIPTION
**Pull request by Rhett-Ying**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: tutorials/blitz/6_load_data.py [hunk 1]
```diff
@@ -222,6 +222,18 @@ def __len__(self):
...
+######################################################################
+# Creating Dataset from CSV via :class:`~dgl.data.DGLCSVDataset`
+# ------------------------------------------------------------
+#
+# In the previous examples, dataset is created directly from raw CSV
+# files via :class:`~dgl.data.DGLDataset`. DGL provides utility class
+# :class:`~dgl.data.DGLCSVDataset` to read data from CSV files and
+# construct :class:`~dgl.DGLGraph` more flexibly. Please refer to
+# :ref:`guide-data-pipeline-loadcsv` and see if this utility is more
+# suitable for your case.
```
Comment (jermainewang) [inline tutorials/blitz/6_load_data.py:234]: The previous examples describe how to create a dataset from CSV files step-by-step.
DGL also provides a utility class :class:`~dgl.data.DGLCSVDataset` for reading and parsing data from CSV files. See :ref:`guide-data-pipeline-loadcsv` for more details.

File: python/dgl/data/csv_dataset.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 
 class DGLCSVDataset(DGLDataset):
-    """ This class aims to parse data from CSV files, construct DGLGraph
-        and behaves as a DGLDataset.
+    """This class aims to parse data from CSV files, construct DGLGraph
+    and behaves as a DGLDataset.
```
Comment (jermainewang) [inline python/dgl/data/csv_dataset.py:10]:
```
Dataset class that loads and parse graph data from CSV files.
```

File: python/dgl/data/csv_dataset.py [hunk 3]
```diff
@@ -42,7 +42,13 @@ class DGLCSVDataset(DGLDataset):
         any available graph-level data such as graph-level feature, labels.
 
     Examples
-    [TODO]: link to a detailed web page.
+    --------
+    ``meta.yaml`` and CSV files are under ``csv_dir``.
```
Comment (jermainewang) [inline python/dgl/data/csv_dataset.py:46]:
```suggestion

```

File: python/dgl/data/csv_dataset.py [hunk 4]
```diff
@@ -42,7 +42,13 @@ class DGLCSVDataset(DGLDataset):
...
+    
+    >>> csv_dataset = dgl.data.DGLCSVDataset(csv_dir)
```
Comment (jermainewang) [inline python/dgl/data/csv_dataset.py:48]:
```suggestion
    >>> csv_dataset = dgl.data.DGLCSVDataset("/path/to/data/folder")
```

File: python/dgl/data/csv_dataset.py [hunk 5]
```diff
@@ -6,8 +6,8 @@
 
 
 class DGLCSVDataset(DGLDataset):
```
Comment (jermainewang) [inline python/dgl/data/csv_dataset.py:8]: Could we call it `CSVDataset` instead of `DGLCSVDataset`?

File: tutorials/blitz/6_load_data.py (hunk 1 above)
Comment (Rhett-Ying) [reply tutorials/blitz/6_load_data.py:234]: accepted.

File: python/dgl/data/csv_dataset.py (hunk 2 above)
Comment (Rhett-Ying) [reply python/dgl/data/csv_dataset.py:10]: accepted.

**Review by jermainewang — APPROVED** [review]
LGTM

**Pull request merged**